src/views/common/GoogleButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GoogleLogin from 'react-google-login';
```

Comment:
#### Nitpick
- Is this really necessary?

Reply:
Doing more research on the implementation, I've come up with a solution that won't require us using additional plugin. I will also make an update regarding that.